### DISCUSSION TITLE
Is there a reason not to have a `podDisruptionBudget` defined in helm-chart for WebServer?

### DISCUSSION THREAD

**Discussion by aquam8** [apache/airflow#15693] [Q&A]
Scheduler and pgbouncer can have their `replicas` configured alongside a `podDisruptionBudget`.

But `webserver` only has `replicas`? I'm happy to add it for webserver but I wondered if there are any reason it's not been done already?

Cheers / Matt

**Comment by bobojobo** [comment]
Seems like webserver has it now. But I'm still missing workers and triggerer on my checklist?
